src/main/java/dan200/computercraft/shared/computer/apis/CommandAPI.java

```diff
@@ -98,6 +99,7 @@ public void shutdown()
             }
             catch( Throwable t )
             {
+                ComputerCraft.log.error( "Error running command.", t );
```

Comment:
Yes, but again you lose the stack trace which is the most valuable part of the error.